doc/changelog.md

```diff
@@ -67,6 +67,19 @@
 
   ```
 
+* Support for including a list of previously excluded modules
+  for Autograph conversion.
+  [(#725)](https://github.com/PennyLaneAI/catalyst/pull/725)
+  
+  Including previuosly excluded modules is now possible:
+
+  ```py
+  @qjit(autograph=True, autograph_include=["excluded_module"])
```

Comment:
@rauletorresc just to double check, does this also work with the following?
```suggestion
  @qjit(autograph=True, autograph_include=["excluded_module.submodule.another_submodule"])
```

Reply:
Yes, `Convert("excluded_module.submodule.another_submodule")` will be called before `DoNotConvert("excluded_module")`